.github/actions/pr_comment/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Responsible for leaving a comment on a PR
+
+inputs:
+  message:
+    type: string
+    required: true
+  run_once:
+    type: boolean
+    default: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: chromatic_inputs
+      shell: bash
+      run: |
+        echo "message=${{ inputs.message }}"
+        echo "run_once=${{ inputs.run_once }}"
+    - name: find_comment
+      id: find_comment
+      uses: peter-evans/find-comment@v2
+      with:
+        issue-number: ${{ github.event.number }}
+        body-includes: ${{ inputs.message }}
+        direction: last
+    - name: add_pr_comment_once
+      if: ${{ fromJSON(inputs.run_once) && !contains(steps.find_comment.outputs.comment-body, inputs.message) }}
```

Comment:
is it worth writing a test to make sure this works since we don't implement it anywhere yet?

Reply:
It does get leveraged by the two workflows I updated, but yeah I think a test is warranted here as well.

Reply:
Tests have been added